runner/src/server/plugins/engine/views/index.html

```diff
@@ -1,11 +1,29 @@
 {% from "error-summary/macro.njk" import govukErrorSummary %}
 {% from "partials/components.html" import componentList %}
+{% from "components/sidebar.html" import Sidebar %}
 
 {% extends 'layout.html' %}
 {% block content %}
 
 {% set gridSize = "full" if components[0].type == 'FlashCard' else "two-thirds" %}
-  <div class="govuk-grid-row">
+<div class="govuk-grid-row"> 
+  {% set count = 0 %} 
+  {% for component in components %}
+    {%if component.type.toLowerCase() == "sidebar" %}
+     {% set count = count + 1 %}
+    {% endif %}    
+  {% endfor %}
+
+  {% set gridSize = "one-half" if count == "2" else "two-thirds" %}
```

Comment:
I don't really like what i have done here. It was the only way i was able to get the gridsize and alignment right.
When using dev tools to set the alignment in the css, i discovered that all sidebars would inherit the alignment regardless if they were unique or not.